galeracluster/source/documentation/galera-parameters.rst

```diff
@@ -278,6 +280,29 @@ on the master are committed.
    wsrep_provider_options="cert.optimistic_pa=NO"
 
 
+.. _`datadir`:
+.. rst-class:: section-heading
+.. rubric:: ``datadir``
+
+.. index::
+   pair: wsrep Provider Options; datadir
+
+Set the path to the database root directory.
+
+.. csv-table::
+   :class: doc-options
+
+   "Default Value", "``/var/lib/mysql/``"
+   "Dynamic", "Yes"
+   "Initial Version", "1.0"
+
+The excerpt below is an example of how this Galera parameter might look in the configuration file:
+
+.. code-block:: ini
+
+   wsrep_provider_options="datadir=/var/lib/mysql/"

```

Comment:
er? where did you get this from? you can't place the datadir in wsrep_provider_options, it is in my.cnf by titself

Reply:
Fixed